packages/epics/src/agreements/plugins/change-entry-method/plugin.tsx

```
@@ -0,0 +1,86 @@
+  },
+  {
+    name: 'Invite Only',
+    value: EntryMethodType.INVITE_ONLY,
+  },
+  {
+    name: 'Token Based',
+    value: EntryMethodType.TOKEN_BASED,
+  },
+];
+
+export const ChangeEntryMethodPlugin = ({}: {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)